operations/iuf/workflows/management_rollout.md

```diff
@@ -242,40 +184,37 @@ Follow the following steps to complete the `management-nodes-rollout` stage.
 
 1. Rebuild the NCN worker nodes. Follow the procedure in section [3.3 NCN worker nodes](#33-ncn-worker-nodes) and then return to this procedure to complete the next step.
 
-1. Configure NCN master and NCN storage nodes.
+1. Personalize NCN storage nodes. Follow the procedure in section [3.4 Personalize NCN storage nodes](#34-personalize-ncn-storage-nodes) and then return to this procedure to complete the next step.
+
+1. Personalize NCN master nodes.
 
-    1. (`ncn-m#`) Create a comma-separated list of the xnames for all NCN master and NCN storage nodes and verify they are correct.
+    1. (`ncn-m#`) Get a comma-separated list of the xnames for all NCN master nodes and verify they are correct.
 
         ```bash
         MASTER_XNAMES=$(cray hsm state components list --role Management --subrole Master --type Node --format json | jq -r '.Components | map(.ID) | join(",")')
-        STORAGE_XNAMES=$(cray hsm state components list --role Management --subrole Storage --type Node --format json | jq -r '.Components | map(.ID) | join(",")')
-        MASTER_STORAGE_XNAMES="${MASTER_XNAMES},${STORAGE_XNAMES}"
         echo "Master node xnames: $MASTER_XNAMES"
-        echo "Storage node xnames: $STORAGE_XNAMES"
-        echo "Master and storage node xnames: $MASTER_STORAGE_XNAMES"
         ```
 
     1. Get the CFS configuration created for management nodes during the `prepare-images` and `update-cfs-config` stages. Follow the instructions in the [`prepare-images` Artifacts created](../stages/prepare_images.md#artifacts-created)
-       documentation to get the value for `configuration` for any image with a `configuration_group_name` value matching `Management_Storage`,`Management_Storage`, or `Management_Storage` (since `configuration` is the same for all
-       management nodes).
+       documentation to get the value for `configuration` for any image with a `configuration_group_name` value matching `Management_Master`.
 
     1. (`ncn-m#`) Set `CFS_CONFIG_NAME` to the value for `configuration` found in the previous step.
 
         ```bash
         CFS_CONFIG_NAME=<appropriate configuration value>
         ```
 
-    1. (`ncn-m#`) Apply the CFS configuration to NCN master nodes and NCN storage nodes.
+    1. (`ncn-m#`) Apply the CFS configuration to NCN master nodes.
 
         ```bash
         /usr/share/doc/csm/scripts/operations/configuration/apply_csm_configuration.sh \
-        --no-config-change --config-name "${CFS_CONFIG_NAME}" --xnames $MASTER_STORAGE_XNAMES --clear-state
+        --no-config-change --config-name "${CFS_CONFIG_NAME}" --xnames $MASTER_XNAMES --clear-state
```

Comment:
I think the docs-csm convention is ```"${MASTER_XNAMES}"```.  This might apply to other areas (I didn't do an exhaustive search).

Reply:
fixed